2] loto/main.py

```diff
@@ -0,0 +1,79 @@
+from random import sample
+import os
+
+def print_fortune(fortune):
+    os.system("clear")
+    for i in range(0, len(fortune), 5):
+        print(*fortune[i:i+5])
+
+def select_number():
+    os.system("clear")
+#    fortune = [i for i in range(1,50)]
+    fortune = list(range(1,50))
+    select_numbers = []
+    count = 0
+
+#    while not count == 6:
+    while count < 6:
+
+       print_fortune(fortune)
+       print("")
+       print(f"Your select numbers are:")
+       print(*select_numbers)
+       select = input(f"Select {count + 1}. number out of 6, from 1 to 49: ")
+
+
+       try:
+           select = int(select)
+
+           if 1 <= select <= 49:
+               if select not in select_numbers:
+                   select_numbers.append(fortune.pop(select - 1))
+                   select_numbers.sort()
```

Comment:
v kazde iteraci sortujes - to neni potreba. sort se muze udelat jednou a na konci